### PR TITLE
codes for handling attendance related queries

### PR DESCRIPTION
**Pull request by SuryaThiru**
the code assumes the db contains 4 subjects, the program worked on my localhost

### CODE REVIEW

File: attendance/database.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import psycopg2
+
+con = psycopg2.connect(dbname = 'vit', user = 'surya2', host = 'localhost', port = 26257)
+con.set_session(autocommit = True)
+
+cur = con.cursor()
+
+def getRow(query):
+    global cur
```
Comment (shubham0704) [inline attendance/database.py]: Add a doc string here.

File: attendance/database.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    global con
+
+    cur.execute(query)
+
```
Comment (shubham0704) [inline attendance/database.py:23]: remove redundant spaces, add spaces to separate some logical steps inside a function

**Comment by shubham0704** [comment]
Well done with your coding style. I see some pep8 format. checkout pep8.

**Comment by SuryaThiru** [comment]
deepinscreenshot20170325002230

deepinscreenshot20170325002321

deepinscreenshot20170325002252

images from the recent test run

**Comment by shubham0704** [comment]
It would be nice if you could add more queries like -
+ can I skip classes on Tuesday .
Come up with more ideas I will merge after deploying travis.

**Comment by ashwini0529** [comment]
@SuryaThiru @shubham0704 
If I ask **Which is my next class?**
It should respond like this : 
`Your next class is Image Processing. You have 76% attendance in that. If you miss this class, you will be debarred. I would suggest you to attend the class.`
Also, a check for 9 pointers.

**Pull request closed without merging**